### DISCUSSION TITLE
my account was being suspended what should i do

### DISCUSSION THREAD

**Discussion by jameshk1108** [community/community#128907] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

My account was suspended last week, so I created a new account to post this message
I already filled out the info on contact support and they gave me ticket #2839452
can someone help me get back my account
my account name is james8611088

please help me

**Comment by ebndev** [comment]
Hi @jameshk1108,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
